### PR TITLE
Extracting viewmodel to protocols

### PR DESCRIPTION
**Pull request by Daniel-Echegaray**
- Extracting viewmodel to protocols for easier seeing its composition
- Extracting Localization keys into LocalizationAssets file
- Creating dataModels used by ViewModel
- Moving View files into View folder

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/Simplified Login/ViewModel/SimplifiedLoginViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+    // MARK: SimplifiedLoginUserActionableSimplified Login User Actions
+    
+    var onClickedContinueWithoutLogin: (() -> Void)?
+    var onClickedSwitchAccount: (() -> Void)?
+    var onClickedPrivacyPolicy: (() -> Void)?
+    var onClickedContinueAsUser: (() -> Void)? // TODO:
+}
+
+extension SimplifiedLoginViewModel {
```
[the remark this answers is not in the document]
Comment (Daniel-Echegaray) [reply Sources/AccountSDKIOSWeb/Simplified Login/ViewModel/SimplifiedLoginViewModel.swift]: That sounds good. I added a new commit with this change. let me know if this i something that you think

File: Sources/AccountSDKIOSWeb/Resources/LocalizationAssets.swift [hunk 2]
```diff
@@ -17,3 +17,41 @@ enum Localization {
         }
     }
 }
+
+struct SimplifiedLoginLocalizationmodel {
```
Comment (zamorastp) [inline Sources/AccountSDKIOSWeb/Resources/LocalizationAssets.swift:21]: Model should start with capital letter

File: Sources/AccountSDKIOSWeb/Simplified Login/ViewModel/SimplifiedLoginViewModel.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (zamorastp) [reply Sources/AccountSDKIOSWeb/Simplified Login/ViewModel/SimplifiedLoginViewModel.swift]: Yes :) good job!

File: Sources/AccountSDKIOSWeb/Resources/LocalizationAssets.swift (hunk 2 above)
Comment (Daniel-Echegaray) [reply Sources/AccountSDKIOSWeb/Resources/LocalizationAssets.swift]: Fixed it 👍

**Review by zamorastp — APPROVED** [review]

**Pull request merged**